src/main/java/bio/terra/service/filedata/google/firestore/FireStoreDao.java

```diff
@@ -81,13 +81,59 @@ public void createDirectoryEntry(Dataset dataset, FireStoreDirectoryEntry newEnt
     directoryDao.createDirectoryEntry(firestore, datasetId, newEntry);
   }
 
+  /**
+   * Write directory entries. If attempting to write a document that has a different ID but the same
+   * load tag, leave the entry untouched and return false for that entry. If the load tags don't
+   * match, then throw an exception
+   *
+   * @param dataset The dataset being ingested into
+   * @param loadTag the load tag of the current ingest
+   * @param directories All directories to upsert
+   * @return A list of booleans the same size as the directories parameter where if an entry is
```

Comment:
derp good call. that was copy pasta. I'll make it return the same conflict map for symmetry (though we don't really need it since this is just for directories)